.github/workflows/CI.yml

```
@@ -0,0 +1,69 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - main
+    paths-ignore:
+      - "README.md"
+      - "LICENSE"
+  pull_request:
+    paths-ignore:
+      - "README.md"
+      - "LICENSE"
+
+jobs:
+  test-save:
+    runs-on: ubuntu-latest
+    outputs:
+      cache-name: ${{ steps.name.outputs.cache-name }}
+    steps:
+    - uses: actions/checkout@v2
```

```suggestion
    - uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579
```

I prefer pinning actions to commit SHAs for security reasons. We can have Dependabot update them if needed.